src/SOS/SOS.Package/SOS.Package.csproj

```diff
@@ -117,4 +117,33 @@
     <ZipDirectory DestinationFile="$(ArtifactsPackagesDir)$(FileVersion).zip" SourceDirectory="$(GalleryDir)" Overwrite="true" />
   </Target>
 
+  <Target Name="GenerateSymbolsZip">
+    <PropertyGroup>
+      <SymbolsDir>$(ArtifactsBinDir)\symbols</SymbolsDir>
+      <SOSNETCorePath>$(ArtifactsBinDir)\SOS.NETCore\$(Configuration)\netstandard2.0\publish</SOSNETCorePath>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <ZipSymbolFiles Include="$(SOSNETCorePath)\*.pdb">
+         <TargetPath>$(SymbolsDir)</TargetPath>
+      </ZipSymbolFiles>
+
+      <ZipSymbolFiles Include="$(ArtifactsBinDir)Windows_NT.x64.$(Configuration)\sos.pdb">
+         <TargetPath>$(SymbolsDir)\win-x64</TargetPath>
+      </ZipSymbolFiles>
+
+      <ZipSymbolFiles Include="$(ArtifactsBinDir)Windows_NT.x86.$(Configuration)\sos.pdb">
+         <TargetPath>$(SymbolsDir)\win-x86</TargetPath>
+      </ZipSymbolFiles>
+
+      <ZipSymbolFiles Include="$(ArtifactsBinDir)Windows_NT.arm.$(Configuration)\sos.pdb">
```

Comment:
Won't we generate a dwarf/dbg artifact, or is it just not needed here?

Reply:
This create Windows only zip files for the gallery.  There was a request from the Windows debugger team for the sos.dll symbols in a separate zip.